array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,13 @@
+/* exported initial */
+function initial(array) {
+  var lastIndex = array.length - 1;
+  var outputArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i === lastIndex) {
+      break;
```

Comment:
Instead of doing this, i would just adjust your condition to stop one iteration sooner.